### DISCUSSION TITLE
Do keyball models support Vial?

### DISCUSSION THREAD

**Discussion by CHBrianLiu** [Yowkees/keyball#556] [Q&A] [answered]
Hi there,

I found these custom keyboard really fantastic. That's what exactly I am looking for! I am curious if it support Vial[^1] to customize the keymap?

[^1]: https://get.vial.today/

Thanks!

**Comment by koron** [comment, marked as answer]
Currently not supported Vial, and there are no plans to support it.

Also considering the firmware size limitations, unfortunately it is hard to support Vial.

**Reply by CHBrianLiu** [reply]
Got it. Thanks @koron!
